main/cmds.go

```diff
@@ -147,15 +155,20 @@ func update(lg ExtensionLogger, hEnv vmextension.HandlerEnvironment, seqNum int)
 		lg.eventError("failed to unzipAgent agent dir", err)
 		return errors.Wrap(err, "failed to unzipAgent agent")
 	}
+	// set permissions for the .sh files
+	err = setPermissions()
```

Comment:
@kwirkykat - Something for us to discuss - What other files need permissions to be changed apart from the `.sh` files.